localization/english/CWP/africa/CWP_libya_l_english.yml

```
@@ -0,0 +1,140 @@
+  jamahiriya_trigger_false_desc_third: "[COUNTRY.GetNameNoFlag] is #b not#! a Jamahiriya of [AddTextIf(ObjectsEqual(GetMetaPlayer.GetPlayedOrObservedCountry.Self, TARGET_COUNTRY.Self),TARGET_COUNTRY.GetName)][AddTextIf(Not(ObjectsEqual(GetMetaPlayer.GetPlayedOrObservedCountry.Self, TARGET_COUNTRY.Self)),TARGET_COUNTRY.GetName)]"
+  jamahiriya_action_propose_name: "Make into Jamahiriya"
+  jamahiriya_action_break_name: "Grant Independence"
+  jamahiriya_proposal_break_accepted_name: "Independence Granted"
+  jamahiriya_proposal_break_accepted_desc: "[TARGET_COUNTRY.GetName] accepted the demand to end our Jamahiriya status. We are independent once again."
+  jamahiriya_proposal_break_declined_name: "Independence Refused"
+  jamahiriya_proposal_break_declined_desc: "[TARGET_COUNTRY.GetName] refused the demand to end our Jamahiriya status."
+  jamahiriya_effect_desc_global: "Becomes a Jamahiriya"
+  jamahiriya_effect_desc_first: "[TARGET_COUNTRY.GetName] becomes a [COUNTRY.GetAdjectiveNoFlag] Jamahiriya"
+  jamahiriya_effect_desc_third: "[AddTextIf(ObjectsEqual(GetMetaPlayer.GetPlayedOrObservedCountry.Self, TARGET_COUNTRY.Self),TARGET_COUNTRY.GetNameNoFlag)][AddTextIf(Not(ObjectsEqual(GetMetaPlayer.GetPlayedOrObservedCountry.Self, TARGET_COUNTRY.Self)),TARGET_COUNTRY.GetName)] becomes a Jamahiriya of [COUNTRY.GetName]"
+
+  #Events
+  libya.1.t: "The Mad Dog of the Middle East"
+  libya.1.d: "In the midst of political turbulence and simmering discontent, Colonel Muammar Gaddafi launches a daring coup attempt to seize control of Libya. Fueled by his unyielding ambition, Gaddafi aspires to rewrite the course of history and establish himself as the undisputed ruler of the nation."
+  libya.1.f: "I am an international leader, the dean of the Arab rulers, the king of kings of Africa and the imam of Muslims, and my international status does not allow me to descend to a lower level.\n― Muammar Gaddafi"
+  libya.1.a: "Form a transitional government"
+  libya.1.b: "Proclaim a socialist state"
+  libya.1.c: "The revolution fails"
+
+  libya.3.t: "Gaddafis Dream"
+  libya.3.d: "In a groundbreaking address to the nation, Libyan leader Muammar Gaddafi announces a momentous decision to centralize and fortify his diverse states into a cohesive and formidable union. With unwavering conviction, Gaddafi sets forth a grand vision that transcends borders, cultures, and historical divisions."
+  libya.3.f: "I am an international leader, the dean of the Arab rulers, the king of kings of Africa and the imam of Muslims, and my international status does not allow me to descend to a lower level.\n― Muammar Gaddafi"
+  libya.3.a: "Gaddafis dream is our dream"
+  libya.3.b: "This is madness"
+
+  libya.4.t: "Gaddafis Influence spreads"
```

- Posessive 's' after Gaddafi
- Capitals in titles
```suggestion
  libya.4.t: "Gaddafi's Influence Spreads"
```